### PR TITLE
SINGA-475 add SoftSign operator

### PR DESCRIPTION
**Pull request by pinpom**

### CODE REVIEW

File: python/singa/autograd.py [hunk 1]
```diff
@@ -1828,3 +1828,26 @@ def backward(self, dy):
...
+    # y = x / (1 + np.abs(x))
+        if training:
+            self.input = x
+        x1 = singa.AddFloat(singa.Abs(x),1.0)
+        y = singa.__div__(x,x1)
+        
+        return y
+​
+    def backward(self, dy):
+        dx = singa.AddFloat(singa.Abs(self.input),1.0)
```
Comment (joddiy) [inline python/singa/autograd.py]: # dy/dx = 1 / ( 1 + |x| ) ^ 2

**Comment by joddiy** [comment]
ready for merge

File: test/python/test_operation.py [hunk 2]
```diff
@@ -610,6 +610,46 @@ def test_Atanh_gpu(self):
         np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT, decimal=5)
         self.check_shape(dx.shape(), (3, 2))
 
+    def test_SoftSign_cpu(self):
+        # y = x / (1 + np.abs(x))
```
Comment (chrishkchris) [inline test/python/test_operation.py]: Could you delete the line 614 since this is the same as line 616? Thanks

**Comment by chrishkchris** [comment]
I have run the test in docker container using cloud K80. It is ready for merge.

**Comment by chrishkchris** [comment]
I have run the test in docker container using cloud K80. It is ready for merge.

**Pull request merged**